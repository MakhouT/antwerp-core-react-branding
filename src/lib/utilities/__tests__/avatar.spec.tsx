import * as React from "react";

import * as A from "../..";
import { renderAndMatchSnapshot } from "../../../__tests__/__fixtures__/testUtils";

describe("Avatar", () => {
    it("Renders default", () => {
        // Arrange
        const element = <A.Avatar
            src="test"
        />;

        // Act and assert
        renderAndMatchSnapshot(element);
    });

    it("Renders small and with name", () => {
        // Arrange
        const element = <A.Avatar
            src="test"
            size={A.Sizes.Small}
            name="me"
        />;

        // Act and assert
        renderAndMatchSnapshot(element);
    });
});
