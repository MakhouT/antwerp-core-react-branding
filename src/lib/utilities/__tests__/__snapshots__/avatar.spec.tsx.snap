// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Avatar Renders default 1`] = `
<div
  className="a-avatar"
>
  <img
    alt=""
    src="test"
  />
</div>
`;

exports[`Avatar Renders small and with name 1`] = `
<div
  className="a-avatar a-avatar--small"
>
  <img
    alt="me"
    src="test"
  />
</div>
`;
